scan/src/pages/ValidatorHomePage.re

```diff
@@ -187,16 +187,39 @@ let renderBody = (rank, validator: ValidatorSub.t) => {
 let make = () =>
   {
     let (page, setPage) = React.useState(_ => 1);
+
+    let (prevDay, setPrevDay) =
+      React.useState(_ =>
+        (
+          MomentRe.momentNow()
+          |> MomentRe.Moment.subtract(~duration=MomentRe.duration(1., `days))
+          |> MomentRe.Moment.toUnix
+          |> float_of_int
+        )
+        *. 1000.
+      );
+    React.useEffect0(() => {
+      let newPrevDay = MomentRe.momentNow() |> MomentRe.Moment.toUnix |> float_of_int;
+      let timeOutId = Js.Global.setTimeout(() => setPrevDay(_ => newPrevDay), 1000);
+      Some(() => Js.Global.clearTimeout(timeOutId));
+    });
+
     let pageSize = 10;
 
+    let lastDay = prevDay |> Js.Json.number;
```

Comment:
No need to declare a new variable. Just use `BlockSub.pastDayCount(prevDay |> Js.Json.number);`